packages/@aws-cdk/aws-pipes-targets-alpha/lib/sagemaker.ts

```diff
@@ -0,0 +1,79 @@
+import { IInputTransformation, IPipe, ITarget, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import { IPipeline } from 'aws-cdk-lib/aws-sagemaker';
+
+/**
+ * SageMaker target properties.
+ */
+export interface SageMakerTargetParameters {
+  /**
+   * The input transformation to apply to the message before sending it to the target.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html#cfn-pipes-pipe-pipetargetparameters-inputtemplate
+   * @default - none
+   */
+  readonly inputTransformation?: IInputTransformation;
+
+  /**
+   * List of parameter names and values for SageMaker Model Building Pipeline execution.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetsagemakerpipelineparameters.html#cfn-pipes-pipe-pipetargetsagemakerpipelineparameters-pipelineparameterlist
+   * @default - none
+   */
+  readonly pipelineParameterList?: SageMakerPipelineParameter[];
```

Comment:
Any reason not to use a map here?
```suggestion
  readonly pipelineParameters?: Record<string, string>;
```